README.md

```diff
@@ -250,7 +250,7 @@ We have added support for the [AWS Kinesis backend](https://docs.netdata.cloud/b
 
 We now have a new, ["text-only" chart type](https://github.com/netdata/netdata/issues/5578), [cpu limits for v2 cgroups](https://github.com/netdata/netdata/issues/5850), [docker swarm metrics](https://docs.netdata.cloud/collectors/go.d.plugin/modules/docker_engine/) and improved [documentation](https://docs.netdata.cloud/).
 
-We continued improving the [Kubernetes helmchart](https://github.com/netdata/helmchart) with liveness probes for slaves, persistence options, a fix for a `Cannot allocate memory` issue and easy configuration for the kubelet, kube-proxy and coredns collectors. 
+We continued improving the [Kubernetes helmchart](https://github.com/netdata/helmchart) with liveness probes for slaves, persistence options, a fix for a `Cannot allocate memory` issue and easy configuration for the kubelet, Kube-proxy and coredns collectors. 
```

Comment:
kube-proxy is correct. 

This is a good [reference](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-proxy/).

Reply:
I would prefer `kube-proxy`. Thanks for pointing this one out.